wrapper.go

```diff
@@ -364,6 +369,20 @@ func (r *resticWrapper) prepareCommand(command string, args *shell.Args, allowEx
 		args.AddArgs(r.profile.GetBackupSource(), shell.ArgConfigBackupSource)
 	}
 
+	// Add retry-lock (supported from restic 0.16, depends on filter being enabled)
+	if lockRetryTime, enabled := r.remainingLockRetryTime(); enabled && filter != nil {
+		// limiting the retry handling in restic, we need to make sure we can retry internally so that unlock is called.
+		lockRetryTime = lockRetryTime - r.global.ResticLockRetryAfter - constants.MinResticLockRetryDelay
+		if lockRetryTime > constants.MaxResticLockRetryTimeArgument {
+			lockRetryTime = constants.MaxResticLockRetryTimeArgument
+		}
+		lockRetryTime = lockRetryTime.Truncate(time.Minute)
+
```

Comment:
I can't figure out why yet, but it seems `containArgument` fails to recognize `--retry-lock` as a valid argument for the `backup` command. Funny enough it works for the `init` command:

```
2023/08/08 22:23:23 resticprofile 0.23.0-dev compiled with go1.20.7
2023/08/08 22:23:23 loading: examples/dev.yaml
2023/08/08 22:23:23 files in configuration are relative to "examples"
2023/08/08 22:23:23 preventing the system from sleeping
2023/08/08 22:23:23 memory available: 13722MB
2023/08/08 22:23:23 setting process priority to 10
2023/08/08 22:23:23 restic 0.16.0
2023/08/08 22:23:23 setting up environment variable: TMP=/tmp
2023/08/08 22:23:23 profile 'self': initializing repository (if not existing)
2023/08/08 22:23:23 unfiltered extra flags:
2023/08/08 22:23:23 unfiltered command: init --cleanup-cache --password-file=examples/key --repo=/Volumes/RAMDisk/self --retry-lock=3m --verbose=1
2023/08/08 22:23:23 setting up environment variable: TMP=/tmp
2023/08/08 22:23:23 starting command: restic init --cleanup-cache --password-file=examples/key --repo=/Volumes/RAMDisk/self --retry-lock=3m --verbose=1
2023/08/08 22:23:23 setting up environment variable: TMP=/tmp
2023/08/08 22:23:23 profile 'self': starting 'backup'
2023/08/08 22:23:23 unfiltered extra flags:
2023/08/08 22:23:23 unfiltered command: backup --cleanup-cache --exclude="/**/.git/" --password-file=examples/key --repo=/Volumes/RAMDisk/self --verbose=1 /go/src/github.com/creativeprojects/resticprofile
2023/08/08 22:23:23 setting up environment variable: TMP=/tmp
2023/08/08 22:23:23 starting command: restic backup --cleanup-cache --exclude="/**/.git/" --password-file=examples/key --repo=/Volumes/RAMDisk/self --verbose=1 /go/src/github.com/creativeprojects/resticprofile
```